spec/unit/provider/service/base_spec.rb

```diff
@@ -2,7 +2,7 @@
 require 'rbconfig'
 require 'fileutils'
 
-describe "base service provider" do
+describe "base service provider", unless: Puppet::Util::Platform.windows? || Puppet::Util::Platform.jruby? do
```

Comment:
I recognize that we are not using these providers in our server, but are other users using JRuby?  Do we support Puppet as a gem running on JRuby?